example/Vagrantfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+ config.vm.box = "ubuntu/xenial64"
+
+ config.vm.provider "virtualbox" do |vb|
+     vb.memory = (1024 * 5).to_s # make sure there's enough memory for all components
+     vb.cpus = 2
+ end
+
+  config.vm.provision "shell", inline: <<-SHELL
+    sudo apt-get update && sudo apt-get install -y apt-transport-https curl ca-certificates curl gnupg-agent software-properties-common make gcc
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    sudo add-apt-repository \
+      "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+      $(lsb_release -cs) \
+      stable"
+    curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+    sudo echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list
+    sudo apt-get update
+    sudo apt-get install -y docker-ce docker-ce-cli containerd.io
```

Comment:
It's easier to use docker provisioner than apt.
You can write the following:

```Vagrantfile
config.vm.provision "docker"
```

Reply:
Interesting. Looking at the docs it does install docker (which I did not know). I'd only argue that it can be confusing to users as - at least in my case - I thought you would provision everything as containers in the VM, when in fact you're only using it do install docker. So, in that case, what do you think of consolidating everything to only one provisioner? I just thought it would read better.

Reply:
I see. I think it's fine as it is.